source/terms.txt

```diff
@@ -0,0 +1,169 @@
+       - "Pass your connection string as the second argument to ``mongo.Connect()``"
+       - "The ``NewColor()`` function takes a ``string`` argument."
+       - `Golang.org reference docs <https://golang.org/ref/spec#Passing_arguments_to_..._parameters>`__
+
+   * - **value** (passed)
+     - | **Description:**
+       | Similar to the `wikipedia definition of value <https://en.wikipedia.org/wiki/Value_(computer_science)>`__ in that it is the "value of a variable" that can be of any data type. This is different from **return values/lists** in that it refers to the contents of the type. See `Go by Example: Values <https://gobyexample.com/values>`__ for examples.
+       | **Recommendation:**
+       | **TODO:** decide whether this term should be formatted in monospace, quotes, or both.
+
+   * - **parameter**
+     - | **Description:**
+       | Identifiers used in method/function signatures. See `Function types <https://golang.org/ref/spec#Function_types>`__ for more info.
+       | **Recommendation:**
+       | Use this term when you are describing the method/function signature.
+       | Example(s):
+
+       - "The ``NewColor()`` method accepts one ``int`` parameter."
+       - "``func RGB(r int, g int, b int)``. This function has three parameters, ``r``, ``g``, and ``b``.
+
+       | When referring to result parameters (also called identifiers), use **result** or **result list**. Example(s):
+
+       - "``func ChangeColor(newColor string) (string, error)``. This function returns a result list containing a ``string`` and ``error``."
+
+   * - **type**
+     - | **Description:**
+       | "A type determines a set of values together with operations and methods specific to those values" (`Types <https://golang.org/ref/spec#Types>`__)
+       | **Recommendation:**
+       | Use this when describing a data type, including categorical (pointer, function, strcut, array, string, etc.). The term "type" should not receive any special formatting, but a specific type name should be monospaced.
+
+   * - **pointer**
+     - | **Description:**
+       | References to values/records. See `Go by Example: Pointers <https://gobyexample.com/pointers>`__ for further explanation and examples.
+       | **Recommendation:**
+       | We can talk about pointers as ``*C`` or "pointer to ``C``". The latter may be easier for a new developer to understand, and to discover the meaning through a google search.
+
+   * - **struct, struct type**
+     - | **Description:**
+       | "A type determines a set of values together with operations and methods specific to those values" (`Struct types <https://golang.org/ref/spec#Struct_types>`__)
+       | An example of a struct:
+
+       .. code-block:: go
+
+          type color struct {
+            name string
+            r,g,b int
+          }
+
+       | **Recommendation:**
+       | Use struct or struct type to refer to both the declaration within a package as well as the data held by a variable.
```

Comment:
Would we be alright drawing a hard line for when one is used over the other?

I would be fine using only "struct type" to  refer to the data held in a variable. And only using "struct" to refer to a declaration within a package.